StripePaymentsUI/StripePaymentsUI/Source/Helpers/STPImageLibrary.swift

```diff
@@ -100,8 +100,7 @@ public class STPImageLibrary: NSObject {
     /// This returns a small icon indicating a card number error for the given card brand.
     @objc(errorImageForCardBrand:) public class func errorImage(for brand: STPCardBrand) -> UIImage
     {
-        let imageName = brand == .amex ? "stp_card_error_amex" : "stp_card_error"
-        return self.safeImageNamed(imageName)
+        return self.safeImageNamed("stp_card_error")
```

Comment:
Is there any reason why we are no longer using the error_amex icon?

https://github.com/stripe/stripe-ios/blob/master/StripePaymentsUI/StripePaymentsUI/Resources/Images/Cards/stp_card_error%403x.png

They look very similar but.. are different:
https://github.com/stripe/stripe-ios/blob/master/StripePaymentsUI/StripePaymentsUI/Resources/Images/Cards/stp_card_error_amex%403x.png

Reply:
We deprecated that image on the web a while ago — we used to have a separate image with 4-6-5 to match Amex's digit grouping instead of the standard 4-4-4-4 grouping. It isn't available in our current list of icons, and it doesn't matter as much these days as card designs are all over the place (e.g. Visa Quick Read)